DepotDownloader/Program.cs

```diff
@@ -94,7 +94,8 @@ static void Main( string[] args )
             ContentDownloader.Config.MaxServers = GetParameter<int>( args, "-max-servers", 20 );
             ContentDownloader.Config.MaxDownloads = GetParameter<int>( args, "-max-downloads", 4 );
             string branch = GetParameter<string>( args, "-branch" ) ?? GetParameter<string>( args, "-beta" ) ?? "Public";
-            var forceDepot = HasParameter( args, "-force-depot" );
+            bool forceDepot = HasParameter( args, "-force-depot" );
+            string os = GetParameter<string>( args, "-os", null );

```

Comment:
As @psychonic said in IRC, this should probably throw an error or something if someone does both `-os` and `-all-platforms`

Reply:
I added an error message now (and fixed the typos). I hope the message is clear enough.